dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -867,6 +867,9 @@ criuEnvDefaults() {
   fi
   export GLIBC_TUNABLES
   export LD_BIND_NOT=on
+  # Disable CRIU's use of HW breakpoints on restore.
+  # HW breakpoints are slower in some environments.
+  export CRIU_FAULT=130 # 130=FI_NO_BREAKPOINTS
```

Comment:
For our current purposes only meaningful states for this var are 1) to be set to `130` (to disable HW breakpoints) and 2) to be unset (defined and empty will be treated as an error by CRIU) to get the default behaviour.

(The variable itself is overloaded, so the other possible numeric values control things unrelated to HW breakpoints.)

If we want someone to be able to override our intention they'll have to be able to tell us that they want `CRIU_FAULT` unset (i.e. HW breakpoints enabled), so we'll need a new variable (e.g. `UNSET_CRIU_FAULT`) since tests like `-z` and `-n` don't distinguish between empty and unset.

We could use a Bash-specific test like `[ -v CRIU_FAULT ]` to avoid the need for a separate variable, but `-v` is Bash-specific and recent, so maybe not for this script?

Reply:
I think we will simply hardcode the value for now.  We can always enhance it later to allow others to define values, if needed.